go.mod

```diff
@@ -209,8 +209,7 @@ replace (
 	cosmossdk.io/api => ./api
 	// use cosmos fork of keyring
 	github.com/99designs/keyring => github.com/cosmos/keyring v1.2.0
-	// celestia-core
-	github.com/cometbft/cometbft => github.com/celestiaorg/celestia-core v1.53.0-tm-v0.38.17-rc2
+	github.com/cometbft/cometbft => github.com/celestiaorg/celestia-core v0.39.10
```

Comment:
[question] did we introduce any changes in v0.39.x that would break compatibility with CometBFT v0.38.x?

Context: celestia-app v5.x currently uses CometBFT v0.38.x and SDK v0.50.x-celestia. If it's safe to bump to v0.39.x here then I can update this table: https://github.com/celestiaorg/celestia-app?tab=readme-ov-file#dependency-branches

Reply:
I was thinking through the same thing, and the logic here for me is simply that we have already ran v0.39.8-10 on mocha running v5 so should be fine

it does have the new reactor etc